mux/node_test.go

```diff
@@ -18,18 +20,108 @@ func TestNewNode(t *testing.T) {
 
 	switch node := node.(type) {
 	case *staticNode:
-		t.Fatalf("Expecting: *mux.staticNode. Wrong node type: %T\n", node)
+		t.Fatalf("Expecting: *mux.regexpNode. Wrong node type: %T\n", node)
 	case *wildcardNode:
-		t.Fatalf("Expecting: *mux.staticNode. Wrong node type: %T\n", node)
+		t.Fatalf("Expecting: *mux.regexpNode. Wrong node type: %T\n", node)
 	}
 
 	node = NewNode("{lang}", 0)
 
 	switch node := node.(type) {
 	case *staticNode:
-		t.Fatalf("Expecting: *mux.staticNode. Wrong node type: %T\n", node)
+		t.Fatalf("Expecting: *mux.wildcardNode. Wrong node type: %T\n", node)
 	case *regexpNode:
-		t.Fatalf("Expecting: *mux.staticNode. Wrong node type: %T\n", node)
+		t.Fatalf("Expecting: *mux.wildcardNode. Wrong node type: %T\n", node)
```

Comment:
why change this ?

Reply:
The error message is incorrect. The expected value should be `wildcard`,but it incorrectly states that the expected value is `static`. This issue is also present in the other error messages.

Reply:
ohh I see 👍🏻